recipe/bld.bat

```diff
@@ -1,4 +1,5 @@
-%PYTHON% setup.py install --single-version-externally-managed --record record.txt
+set PYTHONPATH=".\src"
+%PYTHON% -m pip install . --no-deps --no-build-isolation -vv
```

Comment:
Same story on Windows (though [different part of the conda-build code base]( https://github.com/conda/conda-build/blob/417d3f4c73ba525c037c1f453da24df8b1ee12aa/conda_build/windows.py#L296-L317 ))

```suggestion
%PYTHON% -m pip install . -vv
```